### PR TITLE
Improved utilities/logging documentation

### PR DESCRIPTION
**Pull request by manishbansal8843**
Added some punctuation to improve readability. Also removed some duplicate lines.

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @manishbansal8843,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by manishbansal8843** [comment]
For CLA signing, i am in discussion with my employer. Thank you for the patience.

File: akka-docs/src/main/paradox/typed/logging.md [hunk 1]
```diff
@@ -413,23 +413,6 @@ when the underlying logger implementation is called, which can be surprising at
...
-```
-
-All MDC properties as key-value entries can be included with `%mdc`:
-
-```
-  <encoder>
-    <pattern>%date{ISO8601} %-5level %logger{36} - %msg MDC: {%mdc}%n</pattern>
-  </encoder>
-```
-
```
Comment (patriknw) [inline akka-docs/src/main/paradox/typed/logging.md:432]: ok, this is already shown in the "MDC values" section

**Review by patriknw — APPROVED** [review]
LGTM, after CLA

**Review by octonato — APPROVED** [review]

**Comment by manishbansal8843** [comment]
@patriknw cla is signed. Please merge. Thank you.

**Pull request merged**